Added descriptions of FSRS-related charts + descriptions of new stats (#297)
ankitects/anki-manual: src/stats.md

overdue, so if you have a large backlog, the overdue cards will not be
displayed.

Daily load is an estimate of the average number of cards to be reviewed daily 

> (assuming review at the scheduled time with no backlog). 

Reviewer: Do we need the part in brackets, given the paragraph that follows?

```suggestion
```

overdue, so if you have a large backlog, the overdue cards will not be
displayed.

Daily load is an estimate of the average number of cards to be reviewed daily (assuming review at the scheduled time with no backlog. 

> Daily load = 1/I1 + 1/I2 + 1/I3 +...+ 1/In, where In is the current interval of the n-th card. 

Reviewer 2: We don't want math here, trust me.

> If a card's interval is less than a day, it counts as 1. If you don't have a backlog, daily load should be 

Reviewer 2: This is unnecessary, don't you think. It can also be incorrect if you increase/decrease your daily limits suddenly or if you aren't consistent in studying.

Author: I don't mind removing the math, but I do think that this part is important.

studying is included, it limits how far out the intervals are displayed
to (so 14-month intervals are not displayed at all on a 1-year graph).

**Card Ease**\
This graph shows the number of cards that have a specific ease factor.
Average ease of the selected deck / collection is also displayed here.

**Card Stability**\
Only shown if FSRS is enabled. Stability is defined as the amount of time required
for the probability of recall to decrease from 100% to 90%.

**Card Difficulty**\

> Only shown if FSRS is enabled. Difficulty determines how quickly the intervals grow

Reviewer: How about the following?

```suggestion
Only shown if FSRS is enabled. Difficulty determines how quickly a card's interval grows
```

after each review.

**Card Retrievability**\
Only shown if FSRS is enabled. "Retrievability" is synonymous with "probability of recall".

> Estimated total knowledge is the number of cards the user is expected to know today, 

Reviewer: Expected can read as a requirement ("you are expected to attend the class"), so how about the following instead?

```suggestion
Estimated total knowledge is the number of cards the user is likely to know today, 
```

Other suggestions welcome.

Author: I prefer expected in this case. @brishtibheja thoughts?

Reviewer: How about 'predicted'?

Author: "Predicted to know today" sounds odd to me, granted I'm not a native English speaker

Reviewer: Sounds fine to me, but happy for someone else to weigh in.

Reviewer 2: I think we should use "you" instead of "the user".

As for the verb, "likely" sounds best for the reason dae mentioned. But if we were to use "expected", how about: 
```suggestion
Estimated total knowledge is the number of cards you can expect to know today, 
```

I would still prefer to use:
```suggestion
Estimated total knowledge is the number of cards you are likely to know today, 
```

Author: I'll wait for Dae

Reviewer: Mild preference for 'likely'. One other thought while we're bike-shedding: to avoid any confusion with cards that happen to be due today, one other idea would be something like:

Estimated total knowledge is the number of cards you are likely to currently remember.

Author: Ok, check the latest commit

> calculated as the product of average retrievability and the number of cards that have been 

Reviewer: In slightly more layman's terms:

```suggestion
calculated by multiplying average retrievability and the number of cards that have been 
```
